src/Microsoft.DotNet.Interactive.Tests/Notebook/JupyterNotebookDocumentFileFormatTests.cs

```diff
@@ -549,7 +687,13 @@ public void serialized_code_cells_have_appropriate_shape()
                     {
                         cell_type = "code",
                         execution_count = 1,
-                        metadata = new { },
+                        metadata = new
+                        {
+                            dotnet_interactive = new
```

Comment:
We should represent this metadata with an object, we own this object so we can be more strict

Reply:
I'd argue that this file _shouldn't_ use strict types, because we're testing that we can deserialize JSON of a known shape, and I don't want the test to be "serialize-then-deserialize" because that doesn't properly cover the scenario of "deserialize-from-well-known-shape".  I just went with anonymous types because I didn't want to deal with strings with JSON and all the escaping.

Reply:
I'm not suggesting using a string. Using a defined type instead of an anonymous type has advantages for refactoring, discoverability, test readability, and overall maintenance.

Reply:
Discussed this privately.